code-pushup.config.bundled_oq2x2csd2at.mjs

```diff
@@ -0,0 +1,1210 @@
+// code-pushup.config.ts
+// packages/utils/src/lib/logging.ts
+import isaacs_cliui from '@isaacs/cliui';
+import { cliui } from '@poppinss/cliui';
+// packages/plugin-coverage/src/lib/runner/index.ts
+import { bold } from 'ansis';
+// packages/plugin-coverage/src/lib/nx/coverage-paths.ts
+import { bold as bold2 } from 'ansis';
```

Comment:
This is an artefact from you CP execution that should not get committed

Reply:
oooppps

Reply:
Done